acceptance/tests/aix/aix_package_provider.rb

```diff
@@ -20,7 +20,7 @@ def assert_package_version(package, expected_version)
 
 teardown do
   on hosts, "rm -rf #{dir}"
-  on hosts, puppet('resource', 'package', "'#{package}' ensure=absent")
+  on hosts, "installp -u #{package}"
```

Comment:
I know nothing about AIX so take this with a grain of salt. It looks like puppet's nim provider can either execute [`rpm -e` or `installp -gu`](https://github.com/puppetlabs/puppet/blob/stable/lib/puppet/provider/package/nim.rb#L35-L48). Should we always be using `installp` for puppet-agent packages? Should we be using `-g` to remove dependencies also?